packages/lodestar-cli/src/network/test.js

```diff
@@ -0,0 +1,10 @@
+const {FileENR} = require('./lib/cmds/beacon/fileEnr.js');
```

Comment:
Test file should be converted to a unit test.

Unit test idea:
- create tmp file/path (we have a dependency which helps)
- create a new ENR, then FileENR (either using `new FileENR(...)` or `FileENR.initFromENR(...)`)
- update the enr
- check file for update (read the file into a new ENR, check the contents for the update)
- pass if the file has been written/updated